### ISSUE TITLE
Edit calendar of an event after its creation

### ISSUE THREAD

**Issue by matgardon** [KinToday/kin-web-client#33]
Even though Google Calendar does not allow this, Sunrise and other gcal clients enabled the ability to switch an event to another calendar after it's creation in case you made a mistake and didn't select the calendar you wanted.

I suppose this was done by deleting and recreating the event, and possibly re-sending updates if people were invited to the previous event ?

**Comment by zowesiouff** [comment]
yep, it's an issue that already surfaced in our beta testers group on fb. Like you said, there is no "native" way of doing it via Google APIs: we need to delete the event and recreate it on the other calendar.

For now, we have no ETA for this though, PRs welcomed.
